config/routes.rb

```diff
@@ -4,8 +4,15 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:create, :new]
+      post 'like', on: :member
+      # resources :likes, only: [:create]
```

Comment:
* Kindly remove commented codes from your codebase as this makes your code base bulky an unprofessional 🙏